app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -70,14 +72,21 @@ func (s *URLMapSyncer) EnsureURLMap(lbName string, ing *v1beta1.Ingress, beMap b
 	if err == nil {
 		fmt.Println("url map", name, "exists already. Checking if it matches our desired url map", name)
 		glog.V(5).Infof("Existing url map: %v\n, desired url map: %v", existingUM, desiredUM)
+		// TODO(G-Harmon): Figure out how to properly calculate the fp. Using Sha256 returned a googleapi error.
+		desiredUM.Fingerprint = existingUM.Fingerprint
```

Comment:
Set this in UrlMapMatches?

Reply:
I thought about that, but I didn't like the idea of "urlMapMatches" mutating its parameters. conceptually, they should be constants.

Reply:
Agreed, but urlMapMatches is mutating already.

I think this is different because this is required not really for matching (we could set existingUM.Fingerprint="" for that), but this is required for updating right? Will the update fail if we do not set Fingerprint? Say that in the comment if thats true.